src/components/navbar/navbar.tsx

```diff
@@ -0,0 +1,98 @@
+import { Link, NavLink } from "react-router-dom";
+
+const navbar = () => {
+  return (
+    <div className="navbar flex flex-row justify-between items-center bg-black w-screen h-24 px-24">
+      <div className="container_pages">
+        {/* logo & home link */}
+        <Link
+          to="/"
+          className="
+           text-white
+            cursor-pointer
+            text-2xl
+            -tracking-wider 
+            leading-7 
+            font-noto 
+            font-bold
+            mr-20"
+        >
+          Stellar
+        </Link>
+
+        {/* pages links */}
+        <NavLink
+          to="/planatrip"
+          className={({ isActive }) => {
+            return (
+              "text-white cursor-pointer text-xs leading-7 font-robo font-bold hover:text-red-700 transition-colors duration-200 delay-75 mr-10" +
+              (!isActive ? "" : " text-red-700")
+            );
+          }}
+        >
+          Plan a trip
+        </NavLink>
+        <NavLink
```

Comment:
Nit / Recommended: These are excellent, but if we wanted to add or remove another route we need to scroll through and understand the whole component, that's a little annoying for a team maintaining. this.

It would be super healthy if you had an array structure, that contained the data for our links, and then we rendered them just by iterating over the data which would allow this component to be much smaller and much more readable, something like this:

```
const navigationLinks = [
  {
    to: "/discover",
    label: "Discover"
  },
  {
    to: "/planatrip",
    label: "Plan a trip"
  }
]

{ navigationLinks.map((navigationLink) => {
  return (
     <NavLink
         to={navigationLink.to}
         className="..."
     >
        { navigationLink.label }
    </NavLink>
  )
})}